src/resources/css/tribe-events-full.css

```diff
@@ -679,6 +679,191 @@ select.tribe-events-dropdown {
 	padding-bottom: 0;
 }
 
+/* = Short List View Template Styles
+   @ Template: shortlist.php
+=============================================*/
+.tribe-events-shortlist h2.tribe-events-page-title {
```

Comment:
There's a number of places in the CSS where I see this - and maybe it is modeled after existing CSS in this file (I didn't check), but it is important to note that this:

```css
.tribe-events-shortlist h2.tribe-events-page-title {
```

will perform worse than:

```css
.tribe-events-shortlist .tribe-events-page-title {
```

The benefit of having all of our classes everywhere is so we don't need to specify tags - IDs are faster than Classes which are faster than Tags. One major improvement in this CSS would be to see if you can eliminate the `tag.class` combinations and just go with `.class` instead!